### PR TITLE
Update treebest to latest commit

### PR DESCRIPTION
**Pull request by eparey**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by eparey** [comment]
Hello @bioconda/lib

Any idea why build fails with: make[1]: gcc: Command not found ?
I did not change anything in the build (nor host) requirements, which include - {{ compiler('c') }}.

**Comment by dpryan79** [comment]
The compiler is no longer named `gcc`, which helps ensure that the proper compiler is actually being used. Ensure that the `Makefile` uses the `CC` environment variable.

File: recipes/treebest/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+c_compiler:
+  - gcc      # [linux]
+  - clang    # [osx]
+cxx_compiler:
+  - gxx      # [linux]
+  - clangxx  # [osx]
```
Comment (dpryan79) [inline recipes/treebest/conda_build_config.yaml]: These are the defaults, the binaries just aren't named `gcc` and `clang`.

**Comment by eparey** [comment]
Oh I get it, thanks !!

**Comment by dpryan79** [comment]
Can you encourage the treebest authors to tag a release? We really shouldn't be doing these `postX` pseudo releases on bioconda.

**Comment by eparey** [comment]
Sadly, I don't think they have plans for any new GitHub release...

**Comment by dpryan79** [comment]
I'll ask a contact at Ensembl to request this then.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Comment by dpryan79** [comment]
@eparey As you suspected, there are no future releases for TreeBest expected (I heard back from Ensembl). They're using the last tagged version from 2014 and won't likely ever change. They're also unlikely to ever update TreeBest.

**Pull request merged**